solr/core/src/java/org/apache/solr/update/DocumentBuilder.java

```diff
@@ -336,16 +336,17 @@ private static boolean addCopyFields(
                 + ": "
                 + originalFieldValue);
       }
+      Object fieldValue = originalFieldValue;
```

Comment:
Perhaps `final Object originalFieldValue` could be used on the method signature to show the intention that it is not to be assigned to (any more).

Reply:
fixed in 472e9a9676d3cd26f890cf3c9c6c1bfc9533f3c1